File: Java/Arithmetic Progression/ArithmeticProgression.java
/**
 * Created by @amanmehara on 15/03/2017.
 */
public class ArithmeticProgression {
    public static void main(String[] args) {
        System.out.println("Arithmetic Progression :");
        System.out.println();

        double firstTerm = 24;
        double commonDifference = 7.5;
        int numberOfTerms = 15;

        System.out.println("first term : " + firstTerm);
        System.out.println("common difference : " + commonDifference);
        System.out.println("number of terms : " + numberOfTerms);
        System.out.println();

        //Arithmetic Progression
        System.out.println("Arithmetic Progression : ");
        for (int i = 0 ; i < numberOfTerms ; i++) {
            double ithTerm = firstTerm + i * commonDifference;
            System.out.println(ithTerm);
        }
    }
}

File: C#/Arithmetic Progression/ArithmeticProgression.cs
using System;

namespace Programming
{
    class ArithmeticProgression
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Arithmetic Progression");
            Console.WriteLine();

            double firstTerm = 24;
            double commonDifference = 7.5;
            int numberOfTerms = 15;

            Console.WriteLine("first term: " + firstTerm);
            Console.WriteLine("common difference: " + commonDifference);
            Console.WriteLine("number of terms: " + numberOfTerms);
            Console.WriteLine();

            // Arithmetic Progression
            Console.WriteLine("Arithmetic Progression:");
            for (int i = 0; i < numberOfTerms; i++)
            {
                double ithTerm = firstTerm + i * commonDifference;
                Console.WriteLine(ithTerm);
            }
            Console.WriteLine();
        }
    }
}